foreman/foreman.spec

```diff
@@ -1004,19 +1012,22 @@ exit 0
 %{foreman_rake} db:seed >> %{_localstatedir}/log/%{name}/db_seed.log 2>&1 || :
 %{foreman_rake} apipie:cache:index >> %{_localstatedir}/log/%{name}/apipie_cache.log 2>&1 || :
 (/sbin/service foreman status && /sbin/service foreman restart) >/dev/null 2>&1
+(/sbin/service %{executor_service_name} status && /sbin/service %{executor_service_name} restart) >/dev/null 2>&1
```

Comment:
Isn't this redundant and already covered in the ```%postun``` systemd macro? It makes me think the other sysvinit-style commands can also be replaced with systemd macros.